frontend/src/actions/parcelsActions.tsx

```diff
@@ -0,0 +1,69 @@
+import * as ActionTypes from "constants/actionTypes";
+
+//Parcel List API action
+
+export interface Parcel {
+  id: number,
+  latitude: number,
+  longitude: number
+}
+
+export interface StoreParcelsAction {
+  type: typeof ActionTypes.STORE_PARCEL_RESULTS
+  parcelList: Parcel[]
+}
+
+export const storeParcelsAction = (parcelList:Parcel[]) => ({
+  type: ActionTypes.STORE_PARCEL_RESULTS,
+  parcelList: parcelList,
+});
+
+//Parcel Detail API action
+
+export interface Address {
+  line1: string,
+  line2: string,
+  city: string,
+  province: string,
+  postal: string,
+}
+
+export interface Building {
+  description: string,
+  address: Address,
+  latitude: number,
+  longitude: number,
+  constructionType: string,
+  postal: string,
+  buildingFloorCount: number,
+  buildingPredominateUse: string,
+  buildingTenancy: string,
+  buildingNetBookValue: number
+}
+
+export interface ParcelDetail {
+  id: number,
+  pid: string,
+  localId: number,
```

Comment:
localId has been moved to the building object ans is of type string now

Reply:
updated.